docs/docs/getting-started/installation/gcp.md

```diff
@@ -0,0 +1,202 @@
+| [**GCE**](https://cloud.google.com/kubernetes-engine/docs/concepts/node-images#cos)    | 6 Google Compute Engine Instances                                                                    | [GCE pricing calculation](https://cloud.google.com/products/calculator#id=0394acda-ce3a-4769-a453-9e2ea6ca4291)  | The instances are a part of the `Node Pool` attached to the GKE cluster. The default instance type is: `n1-standard-2`, OS type: `Container-Optimized OS`. | Yes            |
+| [**CLB**](https://cloud.google.com/load-balancing/docs/network/networklb-target-pools) | 1 Cloud Load Balancer                                                                                | [CLB pricing calculation](https://cloud.google.com/products/calculator#id=da086850-b923-475e-a4bc-9edf3d73c9ed)  | Network (target pool-based)Load Balancer created by the ingress controller Kubernetes service                                                              | No             |
+
+[^1]: Options which can be overwritten with flags to the `airy create` command.
+
+## Install
+
+To create the cluster you must setup your local GCP environment, by [configuring
+your local GCP
+profile](https://cloud.google.com/sdk/docs/install-sdk)
+for the GCP account where all the resources will be created.
+
+Download and install the [Airy CLI](cli/introduction.md).
+
+:::warning
+
+If you want to use Airy Core with auto-generated HTTPS certificates, refer to the [Let's Encrypt section](/getting-started/installation/gcp#https-using-lets-encrypt) for customizing your `airy.yaml` file before proceeding.
+
+:::
+
+Now you can run the following command in your Airy workspace, which will create a Kubernetes cluster with `Airy Core` in your Google account:
+
+```bash
+airy create --provider=gcp
+```
+
+This will execute the following actions:
+
+1. Download two Terraform modules inside the `terraform` directory in the workspace. First module is for creating the GKE cluster, the second is for deploying `Airy Core` on that cluster.
+2. Run the `install.sh` bash script inside the `terraform` directory.
+3. Create an GKE cluster in your Google account (applying the `gcp-gke` state).
+4. Deploy `Airy Core` in that cluster (applying the `airy-core` state).
+5. Print URLs for accessing the UI and API.
+
+If you want to customize your `Airy Core` instance please see our [Configuration
+Section](configuration.md).
+
+After the installation, you can also interact with the components of `Airy Core`
+with the [kubectl](https://kubernetes.io/docs/tasks/tools/) command line
+utility. You can find the kubeconfig of your Airy Core instance in
+`WORKSPACE/terraform/kube.conf`.
+
+### Verification
+
+After the installation process, you can verify that all the pods are running with
+
+```
+kubectl get pods --kubeconfig .terraform/kube.conf
+```
+
+#### Setup your DNS
```

Comment:
Perhaps this can be a subsection of `HTTPS using Let's Encrypt`?

Reply:
😱 i was generously deleting things